webserver/views/index.py

```diff
@@ -41,3 +45,33 @@ def faq():
 @index_bp.route("/roadmap")
 def roadmap():
     return render_template("index/roadmap.html")
+
+
+# listen-group    listens                        0   282202115       282202115       0               none
+@index_bp.route("/current-status")
+def current_status():
+
+    load = "%.2f %.2f %.2f" % os.getloadavg()
+
+    process = subprocess.Popen([config.KAFKA_RUN_CLASS_BINARY, "kafka.tools.ConsumerOffsetChecker",
```

Comment:
I'll make a task for adding a proper robots.txt, since we need it anyway.

http://tickets.musicbrainz.org/browse/LB-70